### DISCUSSION TITLE
Importing Issues

### DISCUSSION THREAD

**Discussion by chekle** [code4recovery/12-step-meeting-list#1127] [Q&A] [answered]
Hi there,

Using the latest version of the plugin we’re unable to import our meetings spreadsheet, the page reloads but no meetings are created. Is anyone else having this issue?

wp version 6.2

php version 7.4.7

Thanks in advance.

**Comment by OSAwebsite** [comment]
I am not part of the development team, but wanted to share some thoughts, especially considering time differences with New Zealand. 

First off: it might be a good idea and a time saver to attach an example of the sheet you are using. I could imagine that the development team might want to double check to see if something is off in that file. If you don't want to share the details in this forum, create a test document. Which might be a good idea to do some trouble shooting at your end.

Also: what exactly is going wrong?
Did you have all meetings on your website originally? Or is this a new import?
Or are you saying that you wanted to refresh all meetings with a new import and did the meetings then not show up?

Which import settings did you choose: 

- 'don't delete anything'
- 'delete only the meetings, locations, and groups for the regions present in this CSV'
- 'delete all meetings, locations, groups, districts, and regions'.

To eliminate the obvious:
Create a document with real or test details and try to import a single meeting, to see if that goes through.
If it does, then create a second document with two meetings and try again.
This will help find out if the error is happening for singles or multiples or both.

If an import fails, check the content for hard returns that are not in double quotes or incorrect placement of commas.
If you do this then at least you will have eliminated a syntax error or an error due to unexpected characters in your import file.

I never leave it to the plugin to replace existing meetings. I have seen this go wrong to often due to the content of the meeting information. If there is a meeting I need to replace in full, I tend to delete the old one and then import the new one.

Just some ideas. Again: I am just a TSML user, I am not a developer but maybe these are some ideas you can do some troubleshooting with so that you have more to work with for the support team.

Good luck!
Nicky
Website coordinator Ontario South Al-Anon

**Reply by joshreisner** [reply]
thanks nicky!

to echo the above, please share the sheet you're importing so we can try it on our test site and see if we encounter any issues

**Comment by chekle** [comment]
Sorry for the lack of clarity!

We tried an import from our meetings spreadsheet but when it failed we then tried with the 'example spreadsheet' from the import page but even that didn't work either. 

Below is a video outlining the steps I've taken:
https://www.loom.com/share/527c0efb5516438f8dcd1fd306354577

FYI, there aren't any error messages in the console or in our server logs to help with the troubleshooting process.

Cheers

**Reply by joshreisner** [reply]
hm, strange! not sure why it would not be working for you. also not sure why it looks so different! have there been any customizations to your site recently? 

here's what it looks like for me: https://www.loom.com/share/9f5046f2f30c4bfbaaf7a88935fe4b41

the recent update is not to blame, it didn't make changes to the importer. you're running `3.14.14`?

**Comment by OSAwebsite** [comment]
That will be helpful, I am sure. This is where I will leave it to the development team, because I am not sure that the example sheet is supposed to work. Good luck!

**Reply by chekle** [reply]
No worries, thank you for your help :)

**Comment by joshreisner** [comment, marked as answer]
having slept on it, i'm still very puzzled. maybe try logging out of wp and logging in again? i wonder if something has gone wrong with your user session. it looks from the video like your admin stylesheet is not loading, perhaps other admin features are not as well.

if that doesn't do anything, perhaps try uninstalling / reinstalling the plugin? 

if that still doens' help, maybe try temporarily deactivating plugins? perhaps start with these since they seem security-related:

* adminimize
* ninjafirewall
* restrict-user-access
* wordfence
* wp-2fa

**Reply by joshreisner** [reply]
i would just turn them off long enough to retry refresh the import page and retry importing a CSV

**Reply by joshreisner** [reply]
turns out it was the "adminimize" plugin

**Comment by OSAwebsite** [comment]
Yeah I too noticed that the import options look different on the New Zealand backend. Might such a thing be caused by a theme?

**Reply by joshreisner** [reply]
usually themes dont affect admin pages, but it's possible!

**Comment by chekle** [comment]
Hi Everyone,

Thank you for being so helpful, it's led me to finding the answer! There is a conflict with another plugin, the 'Adminimize' plugin. As soon as I deactivated it, everything worked fine. I added that plugin to our site a couple of weeks ago to lock down the admin for a specific user, so I feel a bit silly but I guess we'll all know if something similar happens again.

Thanks again.

**Reply by joshreisner** [reply]
great!

**Comment by OSAwebsite** [comment]
Good to know. How insidious though. Do t feel silly: a lot of these things
are unpredictable.

On Sun, May 14, 2023 at 16:56 Josh Reisner ***@***.***> wrote:

> turns out it was the "adminimize" plugin
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1127#discussioncomment-5898942>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOIWBF7FAEVIJUR634MDB6LXGFBIZANCNFSM6AAAAAAX63ETAQ>
> .
> You are receiving this because you commented.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/1127/comments/5898942
> @github.com>
>
-- 


-
Yours in service,
Nicky D
Acting Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by OSAwebsite** [comment]
Josh, I was just wondering about these connected issues:
-when an administrator exports the list of meetings, is it in the right syntax and format to be re-imported right away?
-are there any plans to offer visitors of the website a PDF list of meetings per district, or per user selection?

**Reply by joshreisner** [reply]
hi nicky
- yes, should be!
- no, not at this time

**Comment by OSAwebsite** [comment]
Thanks...


-
Yours in service,
Nicky D
Acting Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*
